src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.ts

```diff
@@ -885,4 +887,14 @@ export class InstructorSessionEditPageComponent extends InstructorSessionBasePag
   private deepCopy<T>(obj: T): T {
     return JSON.parse(JSON.stringify(obj));
   }
+
+  private scrollToNewEditForm(): void {
+    setTimeout(() => {
```

Comment:
Might be to late to state this but there is a `this.pageScrollService` to do the dirty work.

Reply:
I can change this in my next UAT PR :P

Reply:
@xpdavid After trying `this.pageScrollService`, I think that the default `scrollTo` animation feels much smoother than `this.pageScrollService`, even with custom `easingLogic`.

I think we should instead convert all the scroll service to use the default ones? Or we can create our own scroll service using the default ones. Reduces unneeded dependencies.

Reply:
> Or we can create our own scroll service using the default ones.

We can do this.